eZ/Publish/Core/QueryType/QueryParameterContentViewQueryTypeMapper.php

```diff
@@ -61,13 +61,22 @@ private function extractParameters(ContentView $contentView)
 
     /**
      * @param ContentView $contentView
-     * @param string $queryParameterValue
+     * @param string|array $queryParameterValue
      *
      * @return mixed
      */
     private function evaluateExpression(ContentView $contentView, $queryParameterValue)
     {
-        if (substr($queryParameterValue, 0, 2) === '@=') {
+        if (is_array($queryParameterValue)) {
```

Comment:
v1.7.1 is afaik ok here if we aim to drop a 6.7.1 patch release, even a beta for ezplatform-com use, in not to long.